### PR TITLE
{bio}[GCC/11.3.0] Exonerate v2.4.0, SNAP v2.0.1

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/e/Exonerate/Exonerate-2.4.0-GCC-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    ('GLib', '2.72.1'),
+]
+
+# parallel build fails
+parallel = 1
+
+runtest = 'check'
+
+sanity_check_paths = {
+    'files': ['bin/%(namelower)s', 'bin/fastaclip', 'bin/fastaoverlap'],
```
Comment (bedroge) [inline easybuild/easyconfigs/e/Exonerate/Exonerate-2.4.0-GCC-11.3.0.eb:39]: Would it be possible to add sanity check commands for these executables?

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 22.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.10.6
See https://gist.github.com/a7a5d780b5ae54e3d7467014f8a5c3d2 for a full test report.

**Review by bedroge — APPROVED** [review]
lgtm

**Comment by bedroge** [comment]
Going in, thanks @smoors!

**Pull request merged**